addons/sp1/src/commands/actions/create_proof.rs

```diff
@@ -24,25 +24,29 @@ lazy_static! {
                     documentation: "The compiled program ELF.",
                     typing: Type::string(),
                     optional: false,
-                    interpolable: true
+                    tainting: true,
+                    internal: false
                 },
                 inputs: {
                     documentation: "The programs's inputs.",
                     typing: Type::array(Type::string()),
                     optional: false,
-                    interpolable: true
+                    tainting: true,
+                    internal: false
                 },
                 verify: {
                     documentation: "Verify proof locally.",
                     typing: Type::bool(),
                     optional: true,
-                    interpolable: true
+                    tainting: true,
+                    internal: false
                 },
                 sp1_private_key: {
                     documentation: "Verify proof locally.",
                     typing: Type::string(),
                     optional: true,
-                    interpolable: true
+                    tainting: true,
```

Comment:
```suggestion
                    tainting: false,
```
If I've already verified successfully locally, then decide on the next one I want to use a key, I don't think setting it should rerun